src/ui/src/pages/app/executions/Index/components/ExecutionsTable.tsx

```diff
@@ -31,7 +31,13 @@ const ExecutionsTable: React.FC = () => {
       dataField: "created_at",
       text: "Run At",
       formatter: (cell: any, row: any) => {
-        return format(new Date(row.created_at), 'eeee, dd MMMM HH:mm:ss');
+        try {
+          const formattedDate = format(new Date(row.created_at), 'eeee, dd MMMM (HH:mm:ss)');
+          return formattedDate;
+        }
+        catch (e) {
+          return row.created_at
+        }
```

Comment:
Ideally it seems like this logic is something that should go into a date formatter helper that takes in the desired row to format and outputs the formatted date. Then the same logic can be used for `time_window_start`, `time_window_end`, and `created_at`. At the very least it would be good to have the format itself created as a constant that you can import to avoid future headaches trying to update the format.

Reply:
Good call, will change